src/api.js

```diff
@@ -106,15 +106,57 @@ export default class API extends mapboxgl.Control {
   }
 
   /**
-   * get a feature collection of features being editted
+   * select a feature for editing
+   * @param {String} id - the drawId of the feature
+   * @returns {Draw} this
+   */
+  select(id) {
+    this._store.edit(id);
+    return this;
+  }
+
+  /**
+   * select all features for editing
+   * @param {String} id - the drawId of the feature
+   * @returns {Draw} this
+   */
+  selectAll() {
+    this._store.getAllIds()
+      .forEach(id => this.select(id));
+    return this;
+  }
+
+  /**
+   * deselect a feature for editing
+   * @param {String} id - the drawId of the feature
+   * @returns {Draw} this
+   */
+  deselect(id) {
+    this._store.commit(id);
+    return this;
+  }
+
+  /**
+   * deselect all features for editing
+   * @param {String} id - the drawId of the feature
+   * @returns {Draw} this
+   */
+  deselectAll() {
+    this._store.getAllIds()
+      .forEach(id => this.commit(id));
+    return this;
+  }
+
+  /**
+   * get a feature collection of features being edited
    * @return {Object} a feature collection of geometries that are in edit mode
    */
-  getEditing() {
+  getSelected() {
```

Comment:
What are you using this for? Seems weird to me to change the api. Are you looking for something to give you the list of feature ids that are in edit mode?